docs/index.md

```diff
@@ -0,0 +1,30 @@
+# Overview
```

Comment:
Not really, because they differ (badges + license in `README.md` vs. lack of them in `docs/index.md`). Ideally it would be to include the latter in the former, but unfortunately, Markdown doesn't offer such possibility (in contrast to .rst) :confused: